openparticle/src.py

```diff
@@ -371,6 +371,10 @@ def __sub__(self, other):
         )
         return self + neg_other
 
+    @property
+    def coeff(self):
+        return np.prod(list(self.op_dict.values()))
```

Comment:
@simonwa7 This will work if you remove `np.prod()` and just `return list(self.op_dict.values())`.